org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/thingTranslator.js

```diff
@@ -1,16 +1,17 @@
 /* jshint esversion: 6 */
 /* jshint node: true */
 'use strict';
+var OpenT2TErrorClass = require('opent2t').OpenT2TError;

```

Comment:
Agreed but didnt know if it would interfere with the actual module export

---
In reply to: [101368120](https://github.com/openT2T/translators/pull/158#discussion_r101368120) [](ancestors = 101368120)